.github/workflows/test-binaries.yml

```diff
@@ -0,0 +1,23 @@
+name: Test Tezos binaries
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  push:
+    branches:
+      - PruStephan/Add_tests_for_packages
+
+jobs:
+  test_binaries:
+    name: Install and test binaries
+    runs-on: [self-hosted, Linux, X64, nix-with-docker]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Test fedora binaries
+        run: ./docker/tests/scripts/test-fedora-binaries.sh
```

Comment:
GA actually doesn't complete this job successfully, it says
```
./docker/tests/scripts/test-fedora-binaries.sh: line 8: jq: command not found
```
The fact that a non-zero exit code is swallowed and is not propagated is a problem